commands/add.js

```diff
@@ -1,13 +1,14 @@
 const fs = require('fs')
-let tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
+const tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
 
 const add = (newTask) => {
   let toDo = {}
+  const id = () => tasks.length !== 0 ? tasks[tasks.length - 1].id + 1 : 1
+  toDo.id = id()
```

Comment:
The `id()` function could be moved outside of the `add()` function body, since it doesn't need access to any of the internal scope of that function. Also, you might want to change the name to indicate that it returns _a new ID_ that is next in the incrementing sequence.

So you could write:

```js
const nextID = () => tasks.length !== 0 ? tasks[tasks.length - 1].id + 1 : 1

const add = (newTask) => {
  let toDo = { id: nextID() }
  // ...
```